src/main/java/com/powsybl/openloadflow/network/impl/AbstractLfBus.java

```diff
@@ -285,7 +285,11 @@ void addLoad(Load load, LfNetworkParameters parameters) {
     }
 
     void addLccConverterStation(LccConverterStation lccCs, LfNetworkParameters parameters) {
-        getOrCreateLfLoad(null, parameters).add(lccCs, parameters);
+        if (!HvdcConverterStations.isHvdcDanglingInIidm(lccCs, parameters)) {
```

Comment:
Because of that, we end the calculation with NaN as p for the converter station for LCC only. @jeandemanged I don't like it very much...